docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+       command: >
+           sh -c "python manage.py migrate &&
+                     python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
+
+   db:
+       image: postgres:14-alpine
+       restart: always
+       env_file:
+           - .env
+       ports:
+           - "5433:5432"
```

Comment:
The PostgreSQL database is exposed on port `5433` externally and mapped to `5432` internally. Make sure this port configuration does not conflict with other services and is correctly referenced in your application settings.